Marlin/temperature.h

```diff
@@ -414,7 +414,7 @@ class Temperature {
       #if ENABLED(AUTO_POWER_CONTROL)
         powerManager.power_on();
       #endif
-      target_temperature[HOTEND_INDEX] = celsius;
+      target_temperature[HOTEND_INDEX] = MIN(celsius, maxttemp[HOTEND_INDEX]);
       #if WATCH_HOTENDS
```

Comment:
No!
This warrants a MAXTEMP error every time you try to heat to maxtemp. You need to subtract a constant in the magnitude of the max expected overshot at that temperature. At other places we used 15°.

Reply:
Ok, I didn't know about that. I've seen some places in the code, where maximum was limited to MAXTEMP - 15C.

So, this is wrong then too? Should be there a BED_MAXTEMP - 15?

```
      static void setTargetBed(const int16_t celsius) {
        #if ENABLED(AUTO_POWER_CONTROL)
          powerManager.power_on();
        #endif
        target_temperature_bed =
          #ifdef BED_MAXTEMP
            MIN(celsius, BED_MAXTEMP)
          #else
            celsius
          #endif
        ;
        #if WATCH_THE_BED
          start_watching_bed();
        #endif
      }

```